sklearn/decomposition/_fastica.py

```diff
@@ -308,17 +308,17 @@ class FastICA(TransformerMixin, BaseEstimator):
 
     Parameters
     ----------
-    n_components : int, optional
+    n_components : int, default=None
         Number of components to use. If none is passed, all are used.
 
-    algorithm : {'parallel', 'deflation'}
+    algorithm : {'parallel', 'deflation'}, default='parallel'
         Apply parallel or deflational algorithm for FastICA.
 
-    whiten : boolean, optional
+    whiten : bool, default=True
         If whiten is false, the data is already considered to be
         whitened, and no whitening is performed.
 
-    fun : string or function, optional. Default: 'logcosh'
+    fun : string or function, default: 'logcosh'
```

Comment:
```suggestion
    fun : string or function, default='logcosh'
```

Reply:
Indeed I missed this one. I've just pushed again